src/kolibri_android/kolibri_utils.py

```diff
@@ -124,7 +126,7 @@ def _monkeypatch_whitenoise():
     from kolibri.utils import kolibri_whitenoise
 
     logger.info("Applying DynamicWhiteNoise workarounds")
-    kolibri_whitenoise.DynamicWhiteNoise = DynamicWhiteNoise
+    kolibri_whitenoise.DynamicWhiteNoise = AndroidDynamicWhiteNoise
```

Comment:
The alternative to monkey patching this is to just define your own WSGI application instead, I would think, which may feel neater? Might be a few more lines of code though.

Reply:
Yeah, I should play with that properly :) The reason I didn't here is because Kolibri's wsgi.py has a fair bit of stuff in it. Looking at it again, it doesn't change very frequently, so not really that scary, and it would indeed make the rest of it simpler.

Reply:
Okay, I gave it a try in #121. I agree, I like it better that way :+1: